cfme/tests/configure/test_vmware_console_settings.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.configure.configuration import VMwareConsoleSupport
+from utils.appliance.implementations.ui import navigate_to
+from utils.version import current_version
+
+
+@pytest.mark.tier(3)
+@pytest.mark.sauce
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+def test_vmware_console_support(appliance):
+    """Tests that the VMware Console Support setting may be changed."""
+    navigate_to(appliance.server, 'Server')
+
+    console_type_widget = VMwareConsoleSupport.vmware_console_form.console_type
+    old_vm_console_type = console_type_widget.first_selected_option_text
+    assert old_vm_console_type, "The default VMware console type should not be empty"
+
+    for new_vm_console_type in VMwareConsoleSupport.CONSOLE_TYPES:
+        console_settings = VMwareConsoleSupport(
+            appliance=appliance,
+            console_type=new_vm_console_type
+        )
+        console_settings.update()
+        console_type_widget = VMwareConsoleSupport.vmware_console_form.console_type
+        cur_vm_console_type = console_type_widget.first_selected_option_text
+        assert cur_vm_console_type == new_vm_console_type
+
+    console_settings = VMwareConsoleSupport(
```

Comment:
Se response here:  https://github.com/ManageIQ/integration_tests/pull/4685#discussion_r119375836

The real thrust other than it just doesn't work is this is done exactly the same as the all the other classes in the cfme/configure/configuration/__init__.py file all of which likely have the same issue.

If you could pass over this requirement I would appreciate it as I have already spent about two days trying to get this to work.   @mshriver has been instrumental in helping me do that.